ci/devserver.sh

```diff
@@ -22,9 +22,8 @@ SRC_DIR="/app/src"
 
 cd "$SRC_DIR/Dashboard"
 
-bundle exec rake watchbg > "$SRC_DIR/rake.log" 2>&1 &
-bundle exec npm install &
-bundle exec npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &
+npm install &
+npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &
```

Comment:
Do we want to run `npm install` and `npm run` in parallel? I understand that this is not new behavior but I am surprised nonetheless

Reply:
How is this resolved? It is causing the issue described above.

Reply:
isnt it fixed now @dlebrero

Reply:
It is not. Running `npm install` and `npm run` at the same time is causing the above issue.

Reply:
does this not run them sequentially? ```npm install
npm run build:dev > "$SRC_DIR/build.dev.log" 2>&1 &```